packages/deployment/lib/message.js

```diff
@@ -81,16 +83,14 @@ function findOutputValue(outputs, key) {
 }
 
 /**
- * Generates a Cumulus Message template for a Cumulus Workflow
+ * Generates a universal Cumulus Message template for a Cumulus Workflow
  *
- * @param {string} name - name of the workflow
- * @param {Object} workflow - the Step Function workflow object
  * @param {Object} config - Kes config object
  * @param {Array} outputs - an list of CloudFormation outputs
  *
  * @returns {Object} a Cumulus Message template
  */
-function generateWorkflowTemplate(name, workflow, config, outputs) {
+function generateWorkflowTemplate(config, outputs) {
```

Comment:
Is this a 'workflow template' anymore?  Should we call it generateMessageTemplate?      Not going to hold up the PR on this, but it feels like we should consider renaming.

Reply:
I feel like this is a specification vs. implementation kind of issue. I consider this a Cumulus Workflow template because it is a Cumulus Message Template populated in such a way as to be used by Cumulus Workflows. Maybe this is a distinction without a difference, because I can't think of a different use case for Cumulus Message Template, but it feels slightly more accurate to call it a workflow template? Workflow message template might be even more accurate, but that's pretty long.

Reply:
Fair.    "There are 3 hard problems in computer science...."